src/Command/DTO.php

```diff
@@ -16,14 +17,20 @@ protected function configure()
         parent::configure();
         $this->setName('dto')
             ->setDescription('Generate DTO class with given properties')
+            ->setDefinition(
```

Comment:
Take a look at the parent(s). There already are fqn and testfqn argument definitions.